src/components/WwwFrame/PromotionalBanner/Banners/GiftBanner.vue

```diff
@@ -22,56 +24,26 @@ export default {
 @import 'settings';
 
 .gift-banner {
-	display: block;
-	color: $kiva-accent-green;
-	background-color: $kiva-darkgreen;
-	font-weight: bold;
-	border-bottom: 1px solid $kiva-navdivider-green;
-
-	&:hover,
-	&:active {
-		color: $kiva-accent-green;
-	}
-
-	.kiva-card-icon {
-		height: rem-calc(30);
-		width: rem-calc(40);
-		transform: rotate(-10deg);
-		margin-top: 0.1875rem;
-		fill: $kiva-accent-green;
-	}
-
-	.content {
-		display: inline-block;
-		margin-left: 0.3125rem;
-
-		@include breakpoint(medium) {
-			transform: translateY(-26%);
+	background-image: url('../../../../assets/images/backgrounds/tipbar-bg-small.jpg');
```

Comment:
~@

Reply:
This is the only think I didn't try. Rejoice!